src/main/webapp/scripts/tutorial-scripts/facebook-template.js

```diff
@@ -0,0 +1,76 @@
+import {tutorialTemplate} from './tutorial-template.js';
+
+const facebookData = {
```

Comment:
Add a comment, like "Facebook tutorial template data. Contains help links for Facebook." (or something like that). Same for the other similar files.

Reply:
Added comments to the top of all tutorial templates.